articles/architecture-scenarios/b2c/user-logout.md

```diff
@@ -0,0 +1,68 @@
+---
+title: User Logout
+description: What to consider for user logout
+toc: true
+topics:
+    - b2c
+    - ciam
+    - logout
+    - sessions
+contentType: concept
+useCase:
+    - user-logout
+---
+# User Logout
+
+[Logout](/logout) is the act of terminating an authenticated session when they're no longer needed, minimizing the likelihood that unauthorized parties can "take over" the session. This is achieved by provisioning a logout option on the user interface you provide to your users.
+
+Multiple types of sessions can be created when a user logs in (e.g., local application sessions, Auth0 session, third-party Identity Provider sessions), and you will need to determine which of these sessions need to be terminated when the user clicks **Logout**.
```

Comment:
Is it worth having a separate paragraph here? Or just append this to the previous paragraph?